src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -198,11 +235,7 @@ function setTooltipContent() {
     tippyInstance.setContent(tooltipContent)
 }
 
-function setCurrentYearAndDispatchToStore(year, removePristineStatus = false) {
-    if (removePristineStatus) {
-        isPristine.value = false
-    }
-    currentYear.value = year
+function dispatchCurrentYearToStore() {
```

Comment:
If already refactoring I would change this function name to `dispatchPreviewYear` or even better `setPreviewYear`.

Note that `previewYear` is used for the time slider and saved in url.

Reply:
Totally agree, I've done that in a later commit (I'll rebase them today)